schemas/com.snowplowanalytics.badrows/schema_violations/jsonschema/1-0-0

```diff
@@ -0,0 +1,167 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a bad row resulting from schema violations",
+  "self": {
+    "vendor": "com.snowplowanalytics.badrows",
+    "name": "schema_violations",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "failure": {
+      "type": "object",
+      "description": "Information regarding the schema violations",
+      "properties": {
+        "timestamp": {
+          "type": "string",
+          "description": "Timestamp at which the failure occurred",
+          "format": "date-time"
+        },
+        "messages": {
+          "type": "array",
+          "description": "List of failure messages associated with the schema violations",
+          "items": {
+            "type": "object",
+            "description": "Individual schema violation",
+            "anyOf": [
+              {
+                "properties": {
+                  "schemaKey": {
+                    "type": "string",
+                    "description": "Schema which caused the failure",
+                    "pattern": "^iglu:[a-zA-Z0-9-_.]+/[a-zA-Z0-9-_]+/[a-zA-Z0-9-_]+/[0-9]+-[0-9]+-[0-9]+$"
+                  },
+                  "instance": {
+                    "type": "string",
+                    "description": "Where the offending data was found",
+                    "maxLength": 128
+                  },
+                  "igluRepositories": {
+                    "type": "array",
+                    "description": "Iglu repositories tried or where the offending schema was found",
+                    "items": {
+                      "type": "string",
+                      "description": "Name of the Iglu repository",
+                      "maxLength": 128
+                    }
+                  },
+                  "message": {
+                    "type": "string",
+                    "description": "Failure message associated with this schema violation",
+                    "maxLength": 256
+                  }
+                },
+                "required": [ "schemaKey", "instance", "igluRepositories", "message" ],
+                "additionalProperties": false
+              },
+              {
+                "properties": {
```

Comment:
can we add `app_id` here ? 
The main intend of this is:
- It's right now hard to determine who are the emitters putting bad data into your schema collection.
To be able to escalate to the appropriate tool / team, I think mainly interesting is the `app_id`
To clarify `event_name` which is referred here as `schemaKey` ?

Reply:
yes there'll be [the full tracker protocol](https://github.com/snowplow/snowplow/wiki/snowplow-tracker-protocol)